setup.sh

```
@@ -6,7 +6,11 @@ echo "Updating git submodules"
 git submodule update --init --recursive --remote
 
 echo "Building rust lib"
+mv libvpuppr/target/debug/deps/libvpuppr.dll libvpuppr/target/debug/deps/libvpuppr.dll.bak || true
```

the backup files don't appear to be restored on error, so I think it would be better to just let the script error out.

In general, I think just letting the script throw an error is more optimal during development

---

I used a bad filename. it's not really bak as in backup, but rather trash or deleteme.

Actually, maybe we should simply delete the old files instead of renaming. I'll see if that works.